packages/gatsby-theme-newrelic/src/components/Overlay.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import Icon from './Icon';
+import Portal from './Portal';
+import NewRelicLogo from './NewRelicLogo';
+import { useTransition, animated } from 'react-spring';
+
+const Overlay = ({ children, onClick, isOpen = false }) => {
```

Comment:
The `onClick` callback here feels a bit oddly named. If I didn't know how the guts of this component was implemented, this would make me think that I'd execute some handler just for clicking on the backdrop. I suspect this is meant to be used for the handler that should be responsible for closing the overlay. That being the case, how about naming this something like `onClose`?